### PR TITLE
Enhancement/#9-synchronization result status in audit log

### PR DESCRIPTION
**Pull request by OtakuDigitChoc**

### CODE REVIEW

File: src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java [hunk 1]
```diff
@@ -17,13 +17,18 @@
 
     @NotNull
     @NotEmpty
+    private Boolean sync;
```
Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java]: Maybe have a more explicit name, like succeeded (or better if you have)

Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java]: And prefer primitive type (boolean) when it is possible (and it should here)

File: src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java [hunk 2]
```diff
@@ -43,6 +48,14 @@ public void setType(String type) {
         this.type = type;
     }
 
+    public Boolean getSync() {
+        return sync;
+    }
+
+    public void setSync(Boolean sync) {
```
Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java]: Do you really need the setter ?

Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java]: The object is fully constructed after calling the constructor and it should not change (immutability is your friend)

File: src/main/java/org/ozwillo/dcexporter/service/CkanService.java [hunk 3]
```diff
@@ -126,7 +126,12 @@ public void updateResourceData(DcModelMapping dcModelMapping, String resource) {
         DateTimeFormatter dateTimeFormatter = ISODateTimeFormat.dateHourMinuteSecondMillis();
         ckanResourceBase.setLastModified(dateTimeFormatter.print(LocalDateTime.now()));
 
-        ckanClient.updateResourceData(ckanResourceBase);
+        try{
+            ckanClient.updateResourceData(ckanResourceBase);
+            return true;
+        }catch(CkanException ckanException){
+            return ckanException.getCkanResponse().isSuccess();
```
Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/service/CkanService.java]: Really ? Does it mean you can have an exception and isSuccess returning true ??

File: src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java (hunk 1 above)
Comment (OtakuDigitChoc) [inline src/main/java/org/ozwillo/dcexporter/model/SynchronizerAuditLog.java]: succeeded it's good ! I do not understand the second comment ... can you explain me more please

File: src/main/java/org/ozwillo/dcexporter/service/CkanService.java (hunk 3 above)
Comment (OtakuDigitChoc) [inline src/main/java/org/ozwillo/dcexporter/service/CkanService.java]: I don't think so

**Pull request merged**